roles/cs.aws-cloudfront/templates/s3-media-distribution-config.json

```diff
@@ -95,7 +95,7 @@
                         ],
                         "Quantity": 3
                     },
-                    "OriginProtocolPolicy": "http-only",
+                    "OriginProtocolPolicy": "https-only",
```

Comment:
We support HTTP for static files. No need to engage encryption here.

Reply:
reverted